packages/qwik/src/core/api.md

```diff
@@ -942,6 +936,12 @@ export const useOn: (event: string | string[], eventQrl: QRL<(ev: Event) => void
 // @alpha
 export const useOnDocument: (event: string | string[], eventQrl: QRL<(ev: Event) => void>) => void;
 
+// @public
+export const useOnVisibleTask$: (first: TaskFn, opts?: UseEffectOptions | undefined) => void;

```

Comment:
just that we don't forget to rename the `UseEffectOption` interface/type as well

Reply:
Done!